wormhole-connect/src/routes/abstracts/relayAbstract.ts

```diff
@@ -0,0 +1,27 @@
+import {
+  ChainId,
+  ChainName,
+  TokenId,
+} from '@wormhole-foundation/wormhole-connect-sdk';
+import { BigNumber } from 'ethers';
+import { ParsedRelayerMessage, ParsedMessage } from 'utils/sdk';
+
+export abstract class RelayAbstract {
+  // swap information (native gas slider)
+  abstract nativeTokenAmount(
+    destChain: ChainName | ChainId,
+    token: TokenId,
+    amount: BigNumber,
+    walletAddress: string,
+  ): Promise<BigNumber>;
+
+  abstract maxSwapAmount(
+    destChain: ChainName | ChainId,
+    token: TokenId,
+    walletAddress: string,
+  ): Promise<BigNumber>;
+
+  abstract tryFetchRedeemTx(
```

Comment:
The other two functions were removed from routeAbstract, but this one is in both - is that intentional?

Reply:
Good catch, should be removed